docs/Reference.md

```diff
@@ -1281,6 +1282,20 @@ Example:
 <WebView autoManageStatusBarEnabled={false} />
 ```
 
+### `setSupportMultipleWindows`
+
+Sets whether the WebView supports multiple windows. See [Android documentation]('https://developer.android.com/reference/android/webkit/WebSettings#setSupportMultipleWindows(boolean)') for more information. 
```

Comment:
Could you mention the vulnerability if they set this to `false`?

Reply:
Sure, good idea!